src/__tests__/db.integration.test.js

```diff
@@ -1,115 +1,41 @@
-
-  let originalLoggerLevel;
-  // =================== END ===================
 
   beforeAll(async () => {
-    process.env.GP2GP_MESSENGER_SERVICE_URL = gp2gpUrl;
-    process.env.GP2GP_MESSENGER_AUTHORIZATION_KEYS = gp2gpAuth;
-    await MessageFragment.truncate();
-    await RegistrationRequest.truncate();
-    await MessageIdReplacement.truncate();
-    await MessageFragment.sync({ force: true });
     await RegistrationRequest.sync({ force: true });
-    await MessageIdReplacement.sync({ force: true });
-
-    originalLoggerLevel = logger.level;
-    logger.level = "warn";
   });
 
   afterAll(async () => {
-    logger.level = originalLoggerLevel;
-    await MessageFragment.sequelize.sync({ force: true });
-    await RegistrationRequest.sequelize.sync({ force: true });
-    await MessageIdReplacement.sequelize.sync({ force: true });
-
-    await ModelFactory.sequelize.close();
+    await RegistrationRequest.truncate();
+    await RegistrationRequest.sync({ force: true });
+    await RegistrationRequest.sequelize.close();
   });
 
-  it('should verify that the database connection pool is able to handle 100 fragments at once', async () => {
+  it('should verify that the database connection pool is able to handle 100 concurrent transactions simultaneously', async () => {
     // given
-    const odsCode = "B85002";
-    const nhsNumber = 9693796047;
-    const numberOfFragments = 100;
-    const messageId = "D2D446C0-B5DC-4434-9D62-863CDB7BE4E7";
-    const conversationId = "AA54D21B-3B4B-4E6D-86DE-98368F46D9F5";
-    const singleFragment = readFile('COPC_IN000001UK01_01', 'equality-test', 'large-ehr-with-external-attachments', 'original');
-    const fragmentMessageIds = Array(numberOfFragments).fill(null).map(() => uuid().toUpperCase());
-    const fragmentsWithMessageIds = {};
-    const dummyBaseUrl = "http://localhost"
-    const sendFragmentEndpoint = "/ehr-out-transfers/fragment";
-    fragmentMessageIds.forEach(fragmentId => fragmentsWithMessageIds[fragmentId] = singleFragment);
+    const numberOfTransactions = 100;
+    const nhsNumber = 1234567890;
+    const odsCode = "B00145";
+    const databaseOperationPromises = [];
 
     // when
-    await createRegistrationRequest(conversationId, messageId, nhsNumber, odsCode);
-
-    for (let oldFragmentMessageId of fragmentMessageIds) {
-      const newFragmentMessageId = oldFragmentMessageId.slice(0, 35) + '0';
-      await createMessageIdReplacement(oldFragmentMessageId, newFragmentMessageId);
-      getFragment.mockResolvedValueOnce((JSON.parse(fragmentsWithMessageIds[oldFragmentMessageId])));
-      updateFragmentMessageId.mockResolvedValueOnce({
-        newMessageId: newFragmentMessageId ,
-        message: JSON.parse(fragmentsWithMessageIds[oldFragmentMessageId])
-      });
+    for (let i = 0; i < numberOfTransactions; i++) {
+      databaseOperationPromises.push(createRegistrationRequest(uuid().toUpperCase(), uuid().toUpperCase(), nhsNumber, odsCode));
```

Comment:
```suggestion
      databaseOperationPromises.push(createRegistrationRequest(conversationId, messageId, nhsNumber, odsCode));
```
I suggest we create variables for `conversationId `and `messageId `and refer to them within this line to remove the ambiguity of what we are passing into the `createRegistrationRequest `function.